ui/src/app/pages/admin/workspace/research-purpose-section.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import { ResearchPurpose } from 'generated/fetch';
+
+import {
+  getSelectedPopulations,
+  getSelectedPrimaryPurposeItems,
+} from 'app/utils/research-purpose';
+
+import { WorkspaceInfoField } from './workspace-info-field';
+
+interface Props {
+  researchPurpose: ResearchPurpose;
+}
+
+export const ResearchPurposeSection = ({ researchPurpose }: Props) => {
```

Comment:
`ResearchPurpose` is the name of the API entity model here, so we can't use it for the component